app/browser/tabs.js

```diff
@@ -336,6 +336,20 @@ const createNavigationState = (navigationHandle, controller) => {
   })
 }
 
+let backgroundProcess = null
+let backgroundProcessTimer = null
+const runScript = (win, debug, script, cb) => {
```

Comment:
Please add a comment for why debug exists (to keep alive for tests I think?)

Reply:
debug was added so that window is not closed and you can view dev tools

Reply:
that's fine just add a comment to explain it thx

Reply:
done